tests/test_generate_report.py

```diff
@@ -11,10 +11,7 @@ class TestGenerateReport(unittest.TestCase):
 
     def test_generate_report(self):
         """Example of how to test the truth of a statement."""
-
-        generate_qc_report("", "qc.pdf")
-        self.assertTrue(os.path.exists("qc.pdf"))
-
-
-if __name__ == "__main__":
-    unittest.main()
+        directory = "F:/npOptoRecordings/691894_2023-10-04_18-03-13"
```

Comment:
If you share the recording folder, I can cut it down to like 1s and we can include it in the repo for testing

Reply:
This is the recording folder on VAST. \\scratch\ephys\temp\691894_2023-10-04_18-03-13